### PR TITLE
[IMP] various: Conversion of mail templates from jinja to qweb.

### PR DESCRIPTION
**Pull request by kga-odoo**
Description of the issue/feature this PR addresses:
- Convert jinja template into qweb
- Task : https://www.odoo.com/web#id=27033&view_type=form&model=project.task&action=327&menu_id=4720
- Pad : https://pad.odoo.com/p/r.8d2749de4844861078fc36312e573e48
## 

I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/data/email_template_data_invoice.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
         <!--Email template -->
         <record id="email_template_edi_invoice" model="mail.template">
             <field name="name">Invoice - Send by Email</field>
-            <field name="email_from">${(object.user_id.email and '%s &lt;%s&gt;' % (object.user_id.name, object.user_id.email) or '')|safe}</field>
+            <field name="email_from">${object.user_id.name} ${('&lt;%s&gt;' % ((object.user_id.email) if object.user_id.email else '')|safe)}</field>
```
Comment (Gorash) [inline addons/account/data/email_template_data_invoice.xml]: What is the purpose of this change?

File: addons/crm/data/mail_template_data.xml [hunk 2]
```diff
@@ -12,9 +12,9 @@
             <field name="auto_delete" eval="True"/>
             <field name="scheduled_date">${(datetime.datetime.now() + relativedelta(days=2)).strftime('%Y-%m-%d %H:%M')}</field>
             <field name="body_html" ><![CDATA[
-
+            <meta charset="utf-8">
```
Comment (Gorash) [inline addons/crm/data/mail_template_data.xml]: Don't need to have a closing slash ?

**Comment by rop-odoo** [comment]
works continue: https://github.com/odoo/odoo/pull/16726
